### PR TITLE
Add test_the_ui_welcoming_block

### LINKED ISSUE #283: [Student's home page] Check the student’s home page welcoming block UI test 1 #1165
https://github.com/ita-social-projects/SpaceToStudy-Client/issues/1165#issue-1902536330

# [TC-ID] : Verify that all required controls are presented on the home page welcoming block

### Priority
Medium

## Description

Verification of the correct display of elements on the student's home page

### Precondition

The website https://s2s-front-stage.azurewebsites.net/ is opened.
The user is logged in as a Student.

## Test Steps
| Step No.      | Step description       | Input data | Expected result   |
|-------------|:-------------|:-----------|:-----|
| 1. | The Home page is opened |            | The search input field with the “What would you like to learn?" placeholder text and the “Find tutor” button are present |

### PR DESCRIPTION
**Pull request by Headless1**

### CODE REVIEW

File: tests/student/home_page.py [hunk 1]
```diff
@@ -46,12 +50,21 @@ def test_search_field_find_by_title(self):
...
-
     def test_student_can_see_tutors_offers_at_the_home_page(self):
         (HomePageStudent(self.driver)
          .get_search_input()
          .click_find_tutor_btn())
         list_of_offers = (ExploreOffersPage(self.driver)
                           .get_list_of_offers_grid_card())
         self.assertIsNotNone(list_of_offers, "There are no offers")
+
+    @allure.testcase(TEST_CASE_283)
```
Comment (lhalam) [inline tests/student/home_page.py]: set value

File: tests/student/home_page.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
+import allure
+
 from SpaceToStudy.ui.pages.explore_offers.explore_offers_page import ExploreOffersPage
 from SpaceToStudy.ui.pages.home_page.home_student import HomePageStudent
 from tests.test_runners import TestRunnerWithStudent
 
+TEST_CASE_283 = "https://github.com/UA-1023-TAQC/SpaceToStudyTA/issues/283"
```
Comment (lhalam) [inline tests/student/home_page.py]: remove this

File: tests/student/home_page.py (hunk 1 above)
Comment (Headless1) [reply tests/student/home_page.py]: done

File: tests/student/home_page.py (hunk 2 above)
Comment (Headless1) [reply tests/student/home_page.py]: done

**Pull request merged**